drafts/use-cases/3.obtaining-events.md

```diff
@@ -0,0 +1,110 @@
+## Entry point
+
+### Story
+As an application user
+I want to display all my events
+So I can find out what awaits me in the nearest future.
```

Comment:
I find that out of place as Hydra is not about events. What about having both?

> ### Story 
> As an API consumer
> I want to be able to retrieve all members of a collection
> So that I can process them further.
> 
> ### Example
> As an application user 
> I want to display all my events 
> So I can find out what awaits me in the nearest future.